### DISCUSSION TITLE
[Q&A] Cannot install Agent Net Inventory on Ubuntu 20.04.5 LTS

### DISCUSSION THREAD

**Discussion by kenakung** [glpi-project/glpi-agent#356] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

```
root@IT-P1127:~# sudo glpi-agent --list-tasks
sudo: glpi-agent: command not found
```

```
root@IT-P1127:~# sudo dpkg -i  glpi-agent-task-network_1.4-1_all.deb
(Reading database ... 269001 files and directories currently installed.)
Preparing to unpack glpi-agent-task-network_1.4-1_all.deb ...
Unpacking glpi-agent-task-network (1:1.4-1) over (1:1.4-1) ...
dpkg: dependency problems prevent configuration of glpi-agent-task-network:
 glpi-agent-task-network depends on glpi-agent (>= 1:1.4-1); however:
  Package glpi-agent is not installed.

dpkg: error processing package glpi-agent-task-network (--install):
 dependency problems - leaving unconfigured
Processing triggers for man-db (2.9.1-1) ...
Errors were encountered while processing:
 glpi-agent-task-network
```

**Comment by g-bougard** [comment]
Hi @kenakung 

as the output of dpkg is telling you, you're missing dependencies. With `dpkg` command, you'll have to install all dependencies by hand... You should better use `apt` command which will try to resolve dependencies, but you need to also install glpi-agent package.

But you can also simply use the [glpi-agent linux perl installer](https://glpi-agent.readthedocs.io/en/latest/installation/index.html#linux-perl-installer) which embeds all the required debian packages from the project and will launch `apt` for you. You only required to have `perl` installed on the system.
